src/dispatcher/src/main/java/org/acme/ControllerResource.java

```diff
@@ -0,0 +1,51 @@
+package org.acme;
+
+import io.quarkus.funqy.knative.events.CloudEvent;
+import io.cloudevents.core.builder.CloudEventBuilder;
+import io.quarkus.funqy.Funq;
+import io.vertx.core.json.JsonObject;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.jboss.logging.Logger;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.net.URI;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+public class ControllerResource {
```

Comment:
Could you please rename it DispatcherFunction instead?

Reply:
57aea1cd7ea32a584da6c6bf6cd6e8ad3bc73e10